From cf0b3a56a828b0a69280086e8c7e13f6e080a6ed Mon Sep 17 00:00:00 2001
From: Dan Liang <dan.liang@intel.com>
Date: Tue, 30 Apr 2013 13:52:46 +0800
Subject: [PORT FROM MAIN] Awesomeplayer: when the clip is not seekable, send out the notification

BZ: 48604

original patch:
R3: 37219(BZ: 24779)

local TS clip is not seekable, when we seek, even it can not be seeked,
we should notifythe app seek complete,so app will update the progressbar.

Change-Id: I9241d6ce9408609109bbb77d50b463f9a2cd8250
Orig-Change-Id: I6f5615400fadcf3a38ab58c46fdff9aa47af3607
Signed-off-by: Weian Chen <weian.chen@intel.com>
Signed-off-by: Dan Liang <dan.liang@intel.com>
---
 media/libstagefright/AwesomePlayer.cpp |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/media/libstagefright/AwesomePlayer.cpp b/media/libstagefright/AwesomePlayer.cpp
index 4c48c15..cd26d10 100644
--- a/media/libstagefright/AwesomePlayer.cpp
+++ b/media/libstagefright/AwesomePlayer.cpp
@@ -1331,6 +1331,8 @@ status_t AwesomePlayer::seekTo(int64_t timeUs) {
     if (mExtractorFlags & MediaExtractor::CAN_SEEK) {
         Mutex::Autolock autoLock(mLock);
         return seekTo_l(timeUs);
+    } else {
+        notifyListener_l(MEDIA_SEEK_COMPLETE);
     }
 
     return OK;
-- 
1.7.4.1

